wallets/dashcore/send-receive.rst

```diff
@@ -147,3 +150,8 @@ addresses**. In the same way, sending addresses can be seen by clicking **Window
 
 You can edit an existing address by right clicking and selecting **Edit**
 address from the context menu. 
+
```

Comment:
This got inserted at the end of the file instead of just below the added text describing use of resend. Please move this up to there.